lib/galaxy/model/__init__.py

```diff
@@ -6987,19 +6989,19 @@ class WorkflowStep(Base, RepresentById):
     workflow_id = Column(Integer, ForeignKey("workflow.id"), index=True, nullable=False)
     subworkflow_id = Column(Integer, ForeignKey("workflow.id"), index=True, nullable=True)
     dynamic_tool_id = Column(Integer, ForeignKey("dynamic_tool.id"), index=True, nullable=True)
-    type = Column(String(64))
+    type: str = Column(String(64))
```

Comment:
That's probably going to conflict types inferred by the sqlalchemy types in 2.0. I'm not sure that's enough reason not to annotate them ...